### PR TITLE
Fixed message when session is expired for Google IAP

### LINKED ISSUE #1677: IAP. Message when session has expired
for IAP authentication
 
### Expected behavior
After the session finished is displayed a message: 

Your session has expired. Please refresh a page to continue. 

And when you push the button "Close" page is refreshing

### Actual behavior
Message is shown:

Your session has expired. Please login again. 
Sign In with:

iap04152019

### PR DESCRIPTION
**Pull request by vlbe**
Fixes #1677

### CODE REVIEW

File: js/components/welcome.html [hunk 1]
```diff
@@ -10,11 +10,23 @@
...
+				<p class="text-danger" data-bind="if: isLoggedIn() && tokenExpired() && !$component.isGoogleIapAuth()">
 					Your session has expired. Please login again.
 				</p>
+				<div data-bind="if: isLoggedIn() && tokenExpired() && $component.isGoogleIapAuth()">
+					<p class="text-danger">
+						Your session has expired. Please refresh a page to continue.
+					</p>
 
-				<div>
+					<div class="form-group" data-bind="if: isLoggedIn() && tokenExpired() && $component.isGoogleIapAuth()">
```
Comment (pavgra) [inline js/components/welcome.html]: Why do you repeat `isLoggedIn() && tokenExpired() && $component.isGoogleIapAuth()` which always will be true?

**Comment by chrisknoll** [comment]
@pavel, please flush the toilet.

**Comment by pavel** [comment]
@chrisknoll Thanks for notifying me, but it does not seem that I'm involved in this project in any way.

https://blog.github.com/2011-03-23-mention-somebody-they-re-notified/

**Comment by chrisknoll** [comment]
That should have been directed at @pavgra. Apologies!

**Pull request merged**